jenkins-pipelines/qa/hydra-cleanup-cloud.jenkinsfile

```diff
@@ -46,11 +46,29 @@ pipeline {
                 }
             }
         }
-        stage('Run hydra clean_azure.py') {
+        stage('Clean AWS') {
```

Comment:
we made sure that if there a failure with one of the stages, it moves to the next one ?